Yearn-agents/Multisigs/src/test.utils.ts

```diff
@@ -0,0 +1,99 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+
+const createOwnerAddedFindingGenerator = (ens: string, owner: string): Finding => 
```

Comment:
all findings have the same severity, type and protocol. I would make a generic finding generator and put the name, description, metadata etc. as parameters.

Reply:
This can be done, but that severities & types are subject to changes and also in this case I prefer clarity/simplicity over the code refactor that you suggest. I am always in favor of the refactor, but this one is a minor one and I wouldn't like to apply that refactor in `utils` where the same happens.

Reply:
added some suggestion for refactor in my comment.